src/components/Footer.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import {
+  FontIcon,
+} from 'material-ui';
+import { muiThemeable } from 'material-ui/styles';
+
+const styles = {
+  wrap: {
+    width: '100%',
+    height: 300,
+    display: 'flex',
+    flexWrap: 'wrap',
+    alignItems: 'center',
+    boxSizing: 'border-box',
+    padding: 60,
+    backgroundColor: 'black',
+    color: 'white',
+  },
+  connect: {
+    flex: 1,
+  },
+  madeby: {
+    flex: 1,
+    textAlign: 'right',
+  },
+  icon: {
+    paddingTop: 20,
+    paddingRight: 30,
+    cursor: 'pointer',
+  },
+}
+
+export default muiThemeable()(Footer);
+
+function Footer (props) {
+  return (
+    <div style={{...styles.wrap, fontFamily: props.muiTheme.fontFamily}}>
+      <div style={styles.connect}>
+        Connect with us:
+        <div style={{whiteSpace: 'nowrap'}}>
```

Comment:
Ok, I hope we'll decide for some rules after pros and cons discussion of inline styling in js